pages/volunteersignup.tsx

```diff
@@ -0,0 +1,424 @@
+                            <CloseButton />
+                        </Flex>
+                        <Text fontWeight="700" fontSize="36px">
+                            {formPageHeaders[pageNum]}
+                        </Text>
+                        <Stack spacing={8}>
+                            <Progress
+                                value={getProgressBarValue(pageNum)}
+                                size="sm"
+                                color={BLUE}
+                                mt={8}
+                                mb={6}
+                            />
+                            {formPages.map((formPage, idx) => {
+                                return (
+                                    <Box
+                                        display={
+                                            pageNum === idx ? null : "none"
+                                        }
+                                    >
+                                        {formPage}
+                                    </Box>
+                                );
+                            })}
+                        </Stack>
+                        {getFormButton()}
+                    </Box>
+                </Center>
+            ) : (
+                <Center>
+                    <VStack mt={120} mb={180} spacing={50}>
+                        <Text fontWeight="700" fontSize="24px" align="center">
+                            Account created successfully
+                        </Text>
+                        <Center maxW={481}>
+                            Your account has been successfully created. Click
+                            the button below to start browsing classes to
+                            volunteer for!
+                        </Center>
+
+                        <Link
+                            _hover={{ textDecoration: "none" }}
+                            _focus={{}}
+                            href="/"
+                        >
+                            <Button
+                                color={"white"}
+                                bg={"#0C53A0"}
+                                px={10}
+                                _hover={{
+                                    bg: "#2C6AAD",
+                                }}
+                                _active={{}}
+                                fontWeight={"200"}
+                                borderRadius={100}
+                            >
+                                Browse Classes
+                            </Button>
+                        </Link>
+                    </VStack>
+                </Center>
+            )}
+        </Wrapper>
+    );
+}
+
+/**
+ * getServerSideProps gets the session before this page is rendered
+ */
+export const getServerSideProps: GetServerSideProps = async (
+    context: GetSessionOptions,
+) => {
+    // obtain the next auth session
+    const session = await getSession(context);
```

Comment:
what exactly is the session being used for in this page?

Reply:
The session is for the wrapper